common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/ExecutorDiskUtils.java

```diff
@@ -23,11 +23,19 @@
 
 import com.google.common.annotations.VisibleForTesting;
 
+import org.apache.commons.lang3.SystemUtils;
 import org.apache.spark.network.util.JavaUtils;
 
 public class ExecutorDiskUtils {
 
-  private static final Pattern MULTIPLE_SEPARATORS = Pattern.compile(File.separator + "{2,}");
+  private static final Pattern MULTIPLE_SEPARATORS;
```

Comment:
Thank you for your first contribution, @pan3793 .
- If we start to touch `src/main`, this is not a test-only PR. So, could you remove `[TEST]` from the PR title?
- Instead of `Fix ExternalShuffleBlockResolverSuite ...`, you had better emphasize your contribution on the main code like `ExecutorDiskUtils`. For example, something like `Support Windows OS in ExecutorDiskUtils` will be more proper.

Reply:
Thanks for your guidance, the PR title has been changed.